### ISSUE TITLE
Unable to set the environment variables through nhost admin console.

### ISSUE THREAD

**Issue by thanikkal** [nhost/nhost#498]
Right now, there appears no way to set the environment variables either through the admin console or either part of deploying through config files?

**Comment by elitan** [comment]
Should be possible in the Nhost Dashboard:

![CleanShot 2022-05-02 at 21 54 57@2x](https://user-images.githubusercontent.com/331818/166316259-b1e9e486-d9b5-42bc-aa05-c0946ff8699c.png)

**Comment by thanikkal** [comment]
<img width="546" alt="image" src="https://user-images.githubusercontent.com/1134742/166319230-1c72a091-2944-4ed7-84b9-288c2e89ddd0.png">

I was talking about setting these pre-existing env variables, not custom ones.

How do i get this to the production system?

**Comment by elitan** [comment]
You're not able to set env vars for Auth [[docs](https://docs.nhost.io/reference/hasura-auth/environment-variables)]. We'll make them available in the coming weeks.

**Comment by thanikkal** [comment]
Thank you for responding, in the meanwhile is there any work around? We are trying to use this for an OTP auth from our app. So would you suggest we wait a week to try this?
Btw, is  nhost/config.yaml is where I can set env vars for testing locally?
Thank You.

**Comment by elitan** [comment]
> So would you suggest we wait a week to try this?

Yes you would have to wait until we've added the settings UI for this in our dashboard.

> Btw, is nhost/config.yaml is where I can set env vars for testing locally?

Yes. You can set env vars via the `nhost/config.yaml` like this:

```yaml
auth:
  email_passwordless_enabled: true
```

That will translate into `AUTH_EMAIL_PASSWORDLESS_ENABLED=true` locally.

**Comment by plmercereau** [comment]
passwordless sms can now be activated from `Users -> Login Settings -> Passwordless SMS`
